### DISCUSSION TITLE
Deleted speechscripts

### DISCUSSION THREAD

**Discussion by ghramsey** [MattFiler/LegendPlugin#153] [General]
Looking at the speech scripts I see there are lots of unused and deleted scripts, but triggering them the sounds are still present.
I know I can edit the speechscript lines and I've been messing with that for LV426 mission.  
 It's cool to be able to change things around or have audio not normally triggered be heard.  
 The lines for Foster to get a sample of the air and one from Meeks asking what's on the walls of the derelict ship are two
 I've never heard in normal game play even though the code exists to trigger them.

There are also some lines like "running low on fuel?" & "ammo's pretty low" also "I'm out of ammo" and a few variations.
Does the game still have code to detect low fuel or low ammo that could be used to trigger those sound events ? 
I kinda like these messages.  I don't know where to look for the ammo status though if it still exists.

While I'm on this general subject. How do I trigger the speechscript labeled with _ALT 
I setup alt_character and tried to trigger that and nothing happens.   Are the _ALT dialog lines still in the game or deleted?

**Comment by MattFiler** [comment]
This same concept goes for animations - there are tonnes left in the game that aren't used! 

I've been thinking about adding pre-filled dropdowns for this sort of thing so that you can more easily select assets that are available in the game. Would this be useful?

RE specific ALT lines, do you have their names? I'll take a look.

**Reply by MattFiler** [reply]
(if pre-filling asset dropdowns would be useful, please feel free to open it as an enhancement issue!)

**Reply by ghramsey** [reply]
By all means if that is possible I will certainly ask for enhancement.

**Comment by ghramsey** [comment]
I wonder what possibilities are for using , unused animations and such?
I imagine if the level editor is finished that those assets could be used there.  

So where would  I find the code to detect low ammo?     I would probably be able to tie the low ammo to a speech or speech script.
Probably would want it randomized for variety.  I am still far to green at deciphering the scripts to locate it without direction.

**Reply by MattFiler** [reply]
There are a lot of unused animations - infact ALL of the cutscenes that got binned in the final story rewrite the game had are still present. I wrote up a list of them all [here](https://avp.fandom.com/wiki/Alien:_Isolation_cutscenes) a while ago. I'd like to get them all playing at some point, but it's tricky to figure out locations for the cameras to get the best angles, as the `.anim` files they used to position the camera in pre-rendered cutscenes didn't ship with the game. It's do-able though!

RE ammo - `Character` has `inventory_set` which you might somehow be able to get an inventory ref from. Alternatively there's `FilterIsinInventory` to check if something is present.

A more complex solution would be to implement something through the Scaleform GFX files - I think you could hook the ammo count value back to a script.

**Comment by Cervicia** [comment]
Where can I find these unused animations? In the `Alien Isolation\DATA\UI\MOVIES` folder are only the used animations. I also searched the Alien Isolation folder for `AYZ_SC` and more `.usm` files, but got no results. I also didnt find a reference to the unused animations in the Commands Editor.

**Reply by MattFiler** [reply]
These are all Havok animations within `ANIMATIONS.PAK`. You can play them in-game using the `CMD_PlayAnimation` entities! They're all named the same way, following the prefix in the Xenopedia article I linked :)

**Comment by ghramsey** [comment]
I wonder if there is deleted code for these objective and dialog reference?

[A1_M1001_RIP_2379]
{Samuels? I've found a trauma kit, but some of its missing. I can't risk coming back without something vital. 
I'm going to look for more supplies.}

[A1_M1001_OBJ_9366] {Find the missing items from the trauma kit
personal monitoring device, hemostatic dressing, plasma cell pack, chest seal}

I think most of these have become the parts of the medkit  ?  "SCJ Injector"  "bonding agent" etc.?

Looks like the objective would have replaced the objective shown in the morgue "find a trauma kit" which 
I always cringe at when I watch new players streaming.   

  It's the final objective but there isn't one to find a way to the main hospital.
